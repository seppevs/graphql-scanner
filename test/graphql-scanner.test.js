const path = require('path');
const graphqlScanner = require('../src/graphql-scanner');

describe('graphql-scanner', () => {
  it('should work', () => {
    const dir = path.join(__dirname, './types');
    const result = graphqlScanner(dir);
    expect(result.typeDefs).toMatchSnapshot();
    expect(result.resolvers).toMatchSnapshot();
  });
});
