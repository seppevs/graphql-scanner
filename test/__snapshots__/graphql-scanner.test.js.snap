// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`graphql-scanner should work 1`] = `
"
      type Query {
        currentUser: User
      }
    
      type TodoList {
        subject: String!
        tasks: [String]
      }
    
      type User {
        firstName: String!
        lastName: String
        apikey: String
        todoLists: [TodoList]
      }
    "
`;

exports[`graphql-scanner should work 2`] = `
Object {
  "Query": Object {
    "currentUser": [Function],
  },
  "TodoList": Object {
    "tasks": [Function],
  },
  "User": Object {
    "todoLists": [Function],
  },
}
`;
